### PR TITLE
playbook: factor out config getter

### PR DESCRIPTION
**Pull request by katia-e**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,36 @@
 
 This repo implements the CSI spec v1.1.0 for [GIG.tech's OpenvCloud](https://gig.tech)
 
+## Ansible role CSI-Driver
+
+OVC CSI driver can be installed with Ansible role [csi-driver](roles/scr-driver)
```
Comment (chrisvdg) [inline README.md]:
```suggestion
OVC CSI driver can be installed with Ansible role [csi-driver](roles/csi-driver)
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,36 @@
...
+
+Role variables:
+
+``` yaml
```
Comment (chrisvdg) [inline README.md]: Maybe add full path of this file so people know exactly where to look for right away,
`roles/csi-driver/vars/main.yaml`?

File: README.md [hunk 3]
```diff
@@ -2,6 +2,36 @@
...
+# required
+server_url: "<G8 url>"
+
+cluster_url: "https://your-g8.gig.tech"
+account: "your-account-name"
+client_jwt: "ItsyoOnline-JWT-token"
+persistent_volume_size: "Size of persistent storage"
+```
+
+Example of the configuration file setting credentials and required config see in [config.env.example](config.env.example).
```
Comment (chrisvdg) [inline README.md]: Add that people need to copy `config.env.example` to `config.env` and add their data in that file, noting that `config.env` is in gitignore and should prevent a push of their credentials to the repo.

File: README.md [hunk 4]
```diff
@@ -2,6 +2,36 @@
...
+``` yaml
+- hosts: localhost
+  vars:
+    state: installed
+  roles:
+    - {role: csi-driver}
+```
+
+Variable `state` defines the action to perform. Takes one of two variables: `installed` and `uninstalled`. Default to `installed`.
+
```
Comment (chrisvdg) [inline README.md:28]: Add command how to execute the playbook.

**Review by chrisvdg — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (katia-e) [reply README.md]: why would a user push anything to the repo?

Comment (chrisvdg) [reply README.md]: The user could be a developer, that's why we add the `config.env.example` and `config.env` in gitignore in the first place, directly editing in `config.env.example` defeats this whole purpose

**Review by chrisvdg — APPROVED** [review]

**Pull request merged**